### PR TITLE
Fb1276

### PR DESCRIPTION
**Pull request by spiresheep**

### CODE REVIEW

**Comment by kranar** [comment]
Put up a demo before you submit the PR or push changes.

**Comment by spiresheep** [comment]
A demo has been uploaded.

File: Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import * as Nexus from 'nexus';
+import {CheckMarkButton} from '.';
+import {DropDownButton} from '../../../components';
+import {EntitlementsPageSizing} from './entitlements_page';
+
+interface Properties {
+  entitlementEntry?: Nexus.EntitlementDatabase.Entry;
+  currencyEntry: Nexus.CurrencyDatabase.Entry;
+  isSecurityActive: boolean;
```
[the remark this answers is not in the document]
Comment (spiresheep) [reply Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx]: It's the variable that lets the row know if the checkmark is green. I thought it was green if the security was active.

[the remark this answers is not in the document]
Comment (kranar) [reply Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx]: Okay, then rename this ```isActive```. This page doesn't involve any securities.

**Pull request merged**